Content.Server/Medical/BiomassReclaimer/BiomassReclaimerSystem.cs

```diff
@@ -153,13 +146,10 @@ private void OnUnanchorAttempt(EntityUid uid, ActiveBiomassReclaimerComponent co
         }
         private void OnAfterInteractUsing(Entity<BiomassReclaimerComponent> reclaimer, ref AfterInteractUsingEvent args)
         {
-            if (!args.CanReach || args.Target == null)
-                return;
-
-            if (!CanGib(reclaimer, args.Used))
-                return;
-
-            if (!TryComp<PhysicsComponent>(args.Used, out var physics))
+            if (!args.CanReach
+                || args.Target == null
+                || !CanGib(reclaimer, args.Used)
+                || !TryComp<PhysicsComponent>(args.Used, out var physics))
```

Comment:
This doesn't seem like it should fail if lacking a PhysicsComponent

Reply:
Actually it does! Biomass is meant to be a 1:1 "1kg of corpse converts to 1kg of biomass, which converts to 1kg of clone", unless the cloning difficulty CVar is set otherwise to make it harder. Physics component is something an entity will literally never not have(it's impossible to not have one, and if an admin deletes the physics component from an entity, it will reappear in the very next tick). With this in mind I only wish that Physics components worked like Transform(uid), so that hitting an entity with !TryComp<PhysicsComponent(uid) won't be necessary. But a guy can dream. 

Anyway, the machine outputs an amount of biomass = to physics.Mass, so that's why it's here. :)